lib/menu.rb

```diff
@@ -0,0 +1,45 @@
+class Menu
```

Comment:
your code is very readable.

Reply:
you work is extremely thorough. A lot of which you had implemented, I wouldn't have thought of. you must have a very good process in breaking the user stories. well done!